src/PageContent/Calendar/Dialog.js

```diff
@@ -16,7 +16,12 @@ class Dialog extends Component {
                     futureEvent={this.props.futureEvent}
                     accountType={this.props.accountType}
                 />
-                <DialogContent />
+                <DialogContent
+                    donatingAgency={'Local Point'}
+                    donatingAgencyContactName={'Andrea Benson'}
+                    donatingAgencyContactPhone={2065436975}
```

Comment:
same as DG content, put this dummy data into the `deliver` object in `PageContainer` and pass it down